python_modules/airline-demo/airline_demo_tests/test_airflow_uncontainerized.py

```diff
@@ -0,0 +1,126 @@
+from collections import defaultdict
+
+from dagster.core.execute_marshalling import execute_marshalling, MarshalledOutput
+from dagster.core.execution import RunConfig, yield_pipeline_execution_context
+from dagster.core.execution_plan import create_execution_plan_core
+from dagster.utils import load_yaml_from_glob_list, mkdir_p, script_relative_path
+
+try:
+    from dagster_airflow.scaffold import _key_for_marshalled_result, coalesce_execution_steps
+except ImportError:
+    pass
+
+from airline_demo.pipelines import (
+    define_airline_demo_download_pipeline,
+    define_airline_demo_ingest_pipeline,
+    define_airline_demo_warehouse_pipeline,
+)
+
+from .marks import airflow
+
+
+def _uncontainerized_pipeline_execute(pipeline, config_object):
```

Comment:
this seems concerns that this is is necessary in a test

Reply:
I can probably decompose the work that the DagsterOperator does so that this test helper is cleaner